packages/xlsx-renderer-cli/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "sxr",
```

Comment:
Is it published already? I think something like @xtoolset/cli looks better

Reply:
yes it is published as `sxr`. I think this short name is fine if someone will use it as a global script.

I created @xtoolset org name and package and I will consider about using it further

Reply:
probably in the future `sxr` become alias of `@xtoolset/xlsx-renderer-cli` or backwards